src/components/proposal/ProposalContainer.tsx

```diff
@@ -76,9 +76,24 @@ const proposalReducer = (
   return draftState;
 };
 
+//Option 1: Change isProposalSubmitted to isProposalEditable and have this be true when proposal status is either DRAFT or EDITABLE_SUBMISSION
+//Option 2: Have the isProposalSubmitted bypassed if the proposal status is EDITABLE_SUBMISSION
 const isProposalSubmitted = (proposal: { submitted: boolean }) =>
   proposal.submitted;
 
+//Is the proposal in draft or edited state? If so set read only to false.
+function isProposalEditable(proposal: ProposalSubsetSubmission) {
```

Comment:
I also think the return values should be swapped, or alternatively this function should be renamed to `isReadOnly`.

Reply:
I think the renaming to `isReadOnly` would make more sense, but I don't know how that will react with the fact that it would be called like:
`isReadOnly: isReadOnly(proposal)`

Whereas incersing the True/False would require `isProposalEditable()` to become `!isProposalEditable()` in some cases?

Reply:
I think having something like `isReadOnly: isReadOnly(proposal)` should be fine, as the first `isReadOnly` is the property of an object while the second is a function, so they're in different scopes.